src/actions/motion.ts

```diff
@@ -434,6 +434,16 @@ class BackSpaceInNormalMode extends BaseMovement {
   }
 }
 
+@RegisterAction
+class BackSpaceInVisualMode extends BaseMovement {
+  modes = [ModeName.Visual, ModeName.VisualBlock];
+  keys = ['<BS>'];
+
+  public async execAction(position: Position, vimState: VimState): Promise<Position> {
+    return position.getLeftThroughLineBreaks();
```

Comment:
It should only go through line breaks if `whichwrap` contains `b`
http://vimdoc.sourceforge.net/htmldoc/options.html#'whichwrap'

Reply:
Done!